module/input/SensorFilter/src/MotionModel.hpp

```diff
@@ -86,28 +83,27 @@ namespace module::input {
                 WX = 10,
                 WY = 11,
                 WZ = 12,
-
-                // omegaTTt_bias
-                BX = 13,
-                BY = 14,
-                BZ = 15,
             };
 
             // Default constructor initialises all vectors to zero, and the quaternion to the identity rotation
             StateVec()
                 : rTWw(Eigen::Matrix<Scalar, 3, 1>::Zero())
                 , vTw(Eigen::Matrix<Scalar, 3, 1>::Zero())
                 , Rwt({1, 0, 0, 0})
-                , omegaTTt(Eigen::Matrix<Scalar, 3, 1>::Zero())
-                , omegaTTt_bias(Eigen::Matrix<Scalar, 3, 1>::Zero()) {}
+                , omegaTTt(Eigen::Matrix<Scalar, 3, 1>::Zero()) {}
 
             // Constructor from monolithic vector representation, normalising the quaternion in the process
             StateVec(const Eigen::Matrix<Scalar, size, 1>& state)
                 : rTWw(state.template segment<3>(PX))
                 , vTw(state.template segment<3>(VX))
                 , Rwt(Eigen::Quaternion<Scalar>(state.template segment<4>(QW)).normalized())
-                , omegaTTt(state.template segment<3>(WX))
-                , omegaTTt_bias(state.template segment<3>(BX)) {}
+                , omegaTTt(state.template segment<3>(WX)) {}
+            template <typename OtherDerived>
```

Comment:
What's the purpose of this second one?

Reply:
Allows a `StateVec` to be constructed from something that is not explicitly a `Eigen::Matrix` (think return type from operations on Eigen matrices)

Reply:
Fair enough. That begs the question though - do we need the first one then?

Reply:
I mean.. it builds and the test passes without the first one soo....?? (Someone smarter than me make the decision)